### PR TITLE
docs: go web app integration guide

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

docs: go web app integration guide (zh-cn)

## Testing

Test locally

### CODE REVIEW

File: packages/console/src/assets/docs/tutorial/integrate-sdk/go-web.mdx [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+  activeIndex={props.activeStepIndex}
+  onButtonClick={() => props.onNext(1)}
+>
+<Alert>
+  The following demonstration is built upon the <a href="https://gin-gonic.com">Gin Web Framework</a>.  
+  You may also integrate Logto into other frameworks by taking the same steps.  
+  In the following code snippets, we assume your app is running on <code>http://localhost:8080</code>.
+</Alert>
+
+Execute in the project root directory:
```
Comment (wangsijie) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/go-web.mdx:17]: What about using `run`

File: packages/console/src/assets/docs/tutorial/integrate-sdk/go-web.mdx [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+		PersistAccessToken: true,
+	}
+
+	// ...
+}`}
+</code>
+</pre>
+
+The `PersistAccessToken` config indicates whether the access token needs to be stored in the session.
+
```
Comment (wangsijie) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/go-web.mdx:177]: We can add some descriptions on why it is set to `true`.

**Review by wangsijie — APPROVED** [review]
Overall LGTM

File: packages/console/src/assets/docs/tutorial/integrate-sdk/go-web.mdx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/assets/docs/tutorial/integrate-sdk/go-web.mdx]: updated.

File: packages/console/src/assets/docs/tutorial/integrate-sdk/go-web.mdx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/assets/docs/tutorial/integrate-sdk/go-web.mdx:177]: Per offline discussion, we don't need to explain why this needs to be `true`. It's straightforward.
https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1662445224089459

**Pull request merged**